Modules/System/Base64 Convert/README.md

```diff
@@ -28,6 +28,53 @@ The string representation, in base-64, of the input string.
 
  Converts the value of the input string to its equivalent string representation that is encoded with base-64 digits.
  
+#### Syntax
+```
+procedure ToBase64(String: Text; TextEncoding: TextEncoding): Text
+```
+#### Parameters
+*String ([Text](https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/methods-auto/text/text-data-type))* 
+
+The string to convert.
+
+*TextEncoding ([TextEncoding](https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/methods-auto/textencoding/textencoding-option))*
+
+The Encoding of the Text. If Windows is selected, ansi (1252) is defaulted unless a codepage is specified.
```

Comment:
The encoding of the text. If Windows is selected, ANSI (1252) will be used by default unless a codepage is specified.

Reply:
Will fix that.